src/v2/Apps/Auction/Components/AuctionAssociatedSale.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box, Image, Text } from "@artsy/palette"
+import { startCase } from "lodash"
+import { createFragmentContainer, graphql } from "react-relay"
+import { RouterLink } from "v2/System/Router/RouterLink"
+import { AuctionAssociatedSale_sale } from "v2/__generated__/AuctionAssociatedSale_sale.graphql"
+
+interface AuctionAssociatedSaleProps {
+  sale: AuctionAssociatedSale_sale
+}
+
+const AuctionAssociatedSale: React.FC<AuctionAssociatedSaleProps> = ({
+  sale,
+}) => {
+  if (!sale.associatedSale) {
+    return null
+  }
+
+  return (
+    <Box maxWidth={400}>
+      <RouterLink to={sale.associatedSale.href} textDecoration="none">
```

Comment:
Technically this link tag collapses; you could forego the box and display="block" here